pkgs/by-name/ln/lngen/package.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchFromGitHub
+, haskellPackages
+, lib
+}:
+with haskellPackages;
```

Comment:
I would not add this at the beginning. How about using just what you need to avoid bringing the whole scope of `haskellPackages`.

E.g.: ```  libraryHaskellDepends = with haskellPackages; [ base syb parsec containers mtl ];```